docs/product-business-resource-planner-guide/src/main/asciidoc/AuthoringPlanningAssets/PhaseConfiguration-section.adoc

```diff
@@ -0,0 +1,17 @@
+[[_optaplanner.phaseConfiguration]]
+= Phase Configuration
+:imagesdir: ../..
+
+Planner splits the solving process into multiple phases.
+Every phase represents a single optimization algorithm run, which consumes a result returned by the previous phase.
+For example, a Construction Heuristic phase is usually placed before a Local Search phase to provide a good initial solution that the Local Search further optimizes.
+
+[NOTE]
+====
+By default, the Solver uses a single Construction Heuristic phase followed by a Local Search phase.
+====
+
+image::Workbench/AuthoringPlanningAssets/phase_configuration.png[align="center"]
+
+Click *Add* to add a new phase. Individual phase elements provide additional configuration options.
+Click *Remove* to remove a specific phase from the Solver configuration.
```

Comment:
This section is missing in the resulting PDF - Installing and configuring Red Hat Business Optimizer.